docs/sources/reference/api/docker_remote_api_v1.19.md

```diff
@@ -220,12 +221,15 @@ Json Parameters:
       container
 -   **ExposedPorts** - An object mapping ports to an empty object in the form of:
       `"ExposedPorts": { "<port>/<tcp|udp>: {}" }`
+-   **VolumeDriver** - A string value containing the volume driver to use, `local` by default.
```

Comment:
I realize we're not in docs-review yet, but how are we going to handle "experimental" features in these parts of the documentation? Should they be included as part of the regular documentation but marked "experimental" somehow? Will there be "conditional" flags for building parts of the docs?

Note that for the "summary/global" description of experimental features, there's this: https://github.com/docker/docker/pull/13303